src/todolist.test.js

```diff
@@ -16,4 +16,19 @@ describe('Test if function are working', () => {
     testToDoTask.removeTask(1);
     expect(testToDoTask.listArray.length).toBe(1);
   });
+
+  test('Test if the changeComplete function is working', () => {
+    testToDoTask.changeComplete(1);
+    expect(testToDoTask.listArray[0].completed).toBe(true);
+  });
+
+  test('Test if Edit function is working', () => {
+    testToDoTask.editTask(1, 'test1');
+    expect(testToDoTask.listArray[0].description).toBe('test1');
+  });
+
+  test('Test  for clear Completed function is working properly', () => {
+    testToDoTask.clearCompleted();
+    expect(testToDoTask.listArray.length).toBe(0);
+  });
```

Comment:
Hello @mckent05, thank you for your feedback. could you kindly suggest the specific test we should add to our test project?

Reply:
Sorry about my late response, you added just one task for the test. I am only suggesting you add more than one task and possibly play around with more jest methods like `toContainEqual` e.t.c. This will help you to thoroughly test your functions and ensure they pass most of the use cases they might be faced with. I hope you understand? 😄